docker/local-universe/add-stub-universe.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Script takes a provided stub-universe json file and extracts it contents into a format usable by the local-universe (or other universe) build processes.
+# Author: Justin Lee (jlee@mesosphere.com)
+# Version: 0.1
+# Example usage (from the universe/docker/local-universe directory)
+#       bash add-stub-universe.sh edgelb.json
+#       bash add-stub-universe.sh edgelb-pool.json
+#       cp -rpv stub-repo/packages/* ../../repo/packages
+set -e 
+set -u
+set -o pipefail
+
+if [[ -z "$1" ]]; then
+    echo "Usage: add-stub-universe.sh <stub-universe-json>"
+    exit 1
+elif [[ ! -f "$1" ]]; then
+    echo "File '$1' does not exist"
+    exit 1
+fi
+
+mkdir -p stub-repo/packages
+
+FILE=$1
+PACKAGES=$(jq -r '.packages[].name' $1)
+
+for PACKAGE in $PACKAGES; do
+echo "Building repo structure for $PACKAGE..."
+LETTER=$(echo $PACKAGE | head -c 1 | tr '[:lower:]' '[:upper:]')
+mkdir -p stub-repo/packages/${LETTER}/${PACKAGE}/0
+
+jq --arg PACKAGE "$PACKAGE" '.packages[] | select(.name == $PACKAGE)' $FILE > stub-repo/$PACKAGE.json
+
+cat stub-repo/$PACKAGE.json | jq '.config'  > stub-repo/packages/${LETTER}/${PACKAGE}/0/config.json
+cat stub-repo/$PACKAGE.json | jq '.resource' > stub-repo/packages/${LETTER}/${PACKAGE}/0/resource.json
+cat stub-repo/$PACKAGE.json | jq '.command' > stub-repo/packages/${LETTER}/${PACKAGE}/0/command.json
```

Comment:
For the 3 commands above you can extract release value from `jq .releaseVersion`. What do you think? So it becomes `stub-repo/packages/${LETTER}/${PACKAGE}/${RELEASE}/....json`

Reply:
Do we actually use releaseVersion?  Does it have to match the actual release version?  My concern is a little bit about what happens if somebody tries to load something with `.releaseVersion 2` and we haven't created a /0 and /1 version in the same repo.

Reply:
I'm not enough of an expert on Universe to know if the it's more preferable to have:

Option 1:
* `/<correct-release-number>`, with no preceding release numbers

Option 2:
* `/0`, but not matching the `.releaseVersion` value

I mean, this script is simple enough that I'm okay with it not covering all use cases.  We just need to get *something* published so that customers in airgapped environments can use stub universe definitions without having to come to support (or me) every time.  Looking for an MVP at the moment.

Reply:
> My concern is a little bit about what happens if somebody tries to load something with .releaseVersion 2 and we haven't created a /0 and /1 version in the same repo.

This case should work. Sounds good to me to merge this change as is and improve it when the use case comes up.